upup/pkg/fi/cloudup/awstasks/vpc.go

```diff
@@ -296,6 +305,14 @@ func (e *VPC) TerraformLink() *terraform.Literal {
 	return terraform.LiteralProperty("aws_vpc", *e.Name, "id")
 }
 
+func (e *VPC) TerraformIpv6CidrLink() *terraform.Literal {
+	shared := fi.BoolValue(e.Shared)
+	if shared {
+		klog.Fatalf("only kops managed VPC is supported: %s", e)
```

Comment:
Is there no Terraform data source that could provide the shared VPC's IPv6 CIDR allocation?